frame/bags-list/src/list/mod.rs

```diff
@@ -582,6 +581,12 @@ impl<T: Config> Bag<T> {
 
 		Ok(())
 	}
+
+	/// Iterate over the nodes in this bag (publicly accessible for testing and benchmarks).
```

Comment:
I think benchmark run with the wasm runtime, so without std. You probably want to add `any(.., feature = runtime-benchmarks)`

Reply:
just used for some remote-ext integration tests in polkadot for now